simplified-app-simplye/src/main/assets/Accounts.json

```diff
@@ -133,14 +133,15 @@
         "supportsSimplyESync": true
     },
     {
+        "authPasscodeAllowsLetters": true,
```

Comment:
I see this all throughout `Accounts.json`, so I'm sure it's not a problem, but I don't actually see anywhere in the code that uses it.

Reply:
I think this was recently replaced by the 'which keyboard should be used" code.